src/cache_memoize/__init__.py

```diff
@@ -112,10 +113,14 @@ def inner(*args, **kwargs):
                     cache.set(cache_key, True, timeout)
                 elif result is not None:
                     cache.set(cache_key, result, timeout)
+                else:
+                    cache.set(cache_key, NONE_PLACEHOLDER, timeout)
                 if miss_callable:
                     miss_callable(*args, **kwargs)
             elif hit_callable:
                 hit_callable(*args, **kwargs)
+            if result == NONE_PLACEHOLDER:
+                result = None
             return result
```

Comment:
I think all of the above can be simplified now. Like this:

```python
def inner(*args, **kwargs):
    cache_key = _make_cache_key(*args, **kwargs)
    if kwargs.pop('_refresh', False):
        result = NONE_PLACEHOLDER
    else:
        result = cache.get(cache_key, NONE_PLACEHOLDER)
    if result is NONE_PLACEHOLDER:
        result = func(*args, **kwargs)
        ...
    elif hit_callable:
        ...
```

Reply:
The only idea of my PR to save something instead on `None` and distinguish it from `None` which we get when key is absent. I am sorry but I don't see how we can achieve this by this simplified variant.

Reply:
See comment above.